src/components/Menu/Menu/hooks/useOnCloseMenu.tsx

```diff
@@ -0,0 +1,21 @@
+import { useCallback } from "react";
+import { CloseMenuOption } from "../MenuConstants";
+
+export default function useOnCloseMenu({
```

Comment:
This is changing this hook's signature, is this hook exported for consumption?

Reply:
No, it's not exported outside. Used only in Menu